libs/orders/src/lib/order-hooks/deal-ticket-query.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from '@apollo/client';
+
+export const DEAL_TICKET_QUERY = gql`
```

Comment:
question (blocking): where is this used? I may just be being blind

Reply:
in multiple places in deal-ticket lib